src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  &__title {
+    padding-inline: 16px;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+
+    position: absolute;
+    top: 206px;
+    transition-duration: $transition;
```

Comment:
Ensure that the variable `$transition` is defined in your SCSS files. If it's not defined, it will cause a compilation error.